staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -126,6 +133,11 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 			if callErr, ok := err.(*webhookutil.ErrCallingWebhook); ok {
 				if ignoreClientCallFailures {
 					klog.Warningf("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
+
+					key := hook.Name + "/" + ValidatingWebhookFailedOpenKeySuffix
+					if err := versionedAttr.Attributes.AddAnnotation(key, "true"); err != nil {
+						klog.Warningf("Failed to set admission audit annotation %s to %s for validating webhook %s: %v", key, "true", hook.Name, err)
```

Comment:
we're already logging a warning on the fail open, no need to log again that we couldn't annotate the audit event

Reply:
Sure, will remove this line.